docs/TRANSLATING.md

```diff
@@ -1,80 +1,289 @@
-# General translation workflow
+# Translation workflow
 
-## Preparing a file with English strings
+DOSBox Staging stores the translations in a `gettext` compatible file format,
+but it does not use the `gettext` library. You can edit them with your favorite
+`*.po` file editor, but please follow the workflow described here instead of
+using tools like `xgettext`.
 
-You need to be sure you have the latest English messages file — you need the
-latest version of DOSBox Staging for this purpose. Do one of the following:
+## Choosing the editor
 
-1. Run DOSBox Staging using the command:
-   `dosbox -lang en -c "config -wl en.lng" -c "exit"`
-2. Inside DOSBox Staging execute the following commands:
-   - `config -set language=en` — to make sure the English language is used
-   - `config -wl en.lng` — to write the language file
-   You can supply a full host OS path instead of simply `en.lng`.
+### ASCII file editors
 
-Any of these will produce a file with the current strings to translate; use it
-as a reference.
+You can update the translations using any ASCII editor with UTF-8 support, like
+**Notepad** (Windows), **Notepad++** (Windows), **TextMate** (macOS),
+**Kate** (Linux, KDE), or **Text Editor** (Linux, GNOME).
 
-## Cleaning up a translation file
+Integrated development environments, like **Visual Studio Code**, should work
+too.
 
-In the same way, you can create a cleaned-up translation file for the given
-language — i.e. for French (`fr`) this would be:
+> **Note**
+>
+> Word processors, like **Microsoft Word** or **Apple Pages**, are not suitable!
 
+### PO file editors
+
+Since the `PO` file format is widespread, several dedicated editors exist, i.e.:
+- **Poedit**, cross-platform
+- **Lokalize**, a KDE tool for Linux
+- **Gtranslator**, a GNOME tool for Linux
+- many web-based tools
+
+For DOSBox Staging translation files, it is best to configure the editor to use
+a monospaced (non-proportional) font:
+
+- **Poedit**\
+_File_ -> _Preferences_ -> _General_ -> _Appearance_ ->
+_Use&nbsp;custom&nbsp;text&nbsp;field&nbsp;font_
+- **Lokalize**\
+_Settings_ -> _Configure&nbsp;Lokalize..._ -> _Appearance_ ->
+_Editor&nbsp;font_ -> _Show&nbsp;only&nbsp;monospaces&nbsp;fonts_
+- **Gtranslator**\
+hamburger&nbsp;menu -> _Preferences_ -> _Editor_ -> _Font_
+
+## Adding new translation
+
+To start a new translation, execute the following commands inside the current
+alpha build of DOSBox Staging:
+
+```bat
+config -set language=en
+config -wl xx.po
 ```
-dosbox -lang fr -c "config -wl fr.lng" -c "exit"
+
+where _xx_ is a language code. If only possible, use a two-letter code
```

Comment:
Maybe be more assertive and make this mandatory? "Use the two-letter code according to..."